groups/iam/main.tf

```diff
@@ -0,0 +1,26 @@
+provider "aws" {
+  region  = var.region
+  version = "~> 2.0"
```

Comment:
So I would probably rename this group something like `provisioning-role` to give more context to it. As it stands people may wonder why not just include it under the other group. I.e. tell them what it's for

Reply:
Done. Renamed the group from 'iam' to 'provisioning-role'.